homeassistant/components/octoprint/button.py

```diff
@@ -30,6 +30,8 @@ async def async_setup_entry(
             OctoprintResumeJobButton(coordinator, device_id, client),
             OctoprintPauseJobButton(coordinator, device_id, client),
             OctoprintStopJobButton(coordinator, device_id, client),
+            OctoprintConnectButton(coordinator, device_id, client),
```

Comment:
The feedback from the codeowner is to not include a button for connect, but just a button for disconnect. Please adjust the PR.